lib/player.rb

```diff
@@ -0,0 +1,25 @@
+class Player 
+  include Interface
+
+  attr_reader :char
+  attr_accessor :name, :inputs
+
+  def initialize(name, inputs=[], char)
+    @name = name
+    @inputs = inputs
+    @char = char
+  end
+
+  def initial  
```

Comment:
Guys, please enlighten me on this. What is the purpose of this `initial` method considering we already have the `initialize` method already doing a good job initializing instance variables for us?

Reply:
same answer above.